src/components/Geolocation/Geolocation.js

```diff
@@ -66,10 +72,10 @@ class Geolocation extends PureComponent {
       source: new VectorSource(),
     });
 
-    this.isCentered = true;
+    this.recenterToPostion = true;
```

Comment:
Can we rename this property to `isRecenteringToPosition`  otherwise we could think it's a function.

Reply:
sure